flink-core/src/test/java/org/apache/flink/types/parser/ParserTestBase.java

```diff
@@ -407,26 +407,47 @@ public void testStaticParseMethodWithInvalidValues() {
 	@Test
 	public void testEmptyFieldInIsolation() {
 		try {
-			String [] emptyStrings = new String[] {"|"};
+			FieldParser<T> parser = getParser();
+
+			byte[] bytes = "|".getBytes(ConfigConstants.DEFAULT_CHARSET);
```

Comment:
this is the same case as `"a|"` in `testTailingEmptyField()`. We only need one of these.

Reply:
Yes, we can keep `testEmptyFieldInIsolation`.